src/poly/poly.jl

```diff
@@ -193,20 +193,34 @@ function order(p::spoly)
 end
 
 @doc Markdown.doc"""
-    lead_exponent(p::spoly)
+    leading_exponent_vector(p::spoly)
 
 Return the exponent vector of the leading term of the given polynomial. The return
 value is a Julia 1-dimensional array giving the exponent for each variable of the
 leading term.
 """
-function lead_exponent(p::spoly)
+function leading_exponent_vector(p::spoly)
    R = parent(p)
    n = nvars(R)
    A = Array{Int}(undef, n)
    libSingular.p_GetExpVL(p.ptr, A, R.ptr)
    return A
 end
 
+function lead_exponent(p::spoly)
+   @warn "please use leading_exponent_vector instead of lead_exponent"
```

Comment:
Should this be a deprecation, using `@deprecate`?

Reply:
I do not understand `@deprecate` and recently have had problems with it hanging. I would like to keep it this way and then eventually remove it when it is surely gone from all of the other packages.

Reply:
Let me give it a try.